runbot_gitlab_minimal/models/runbot_repo.py

```diff
@@ -17,12 +17,18 @@
 def _get_url(url, base):
     match_object = re.search('([^/]+)/([^/]+)/([^/.]+(.git)?)', base)
     if match_object:
+        prefix = ('https://%s/api/v3%s'
```

Comment:
Add a comment because if using this replace...

`Because gitlab api v3 requires admin to get keys from api but...`